web/elm/src/Build/Output/Output.elm

```diff
@@ -145,6 +145,11 @@ handleEvent event ( model, effects ) =
             , effects
             )
 
+        SelectedWorker origin output time ->
+            ( updateStep origin.id (setRunning << appendStepLog ("selected worker: " ++ output ++ "\n") time) model
```

Comment:
Might be nice to bold this as we do with the `fly watch` output. Should be pretty much the same, albeit with a different escape char:

```suggestion
            ( updateStep origin.id (setRunning << appendStepLog ("\u{001b}[1mselected worker: \u{001b}[0m" ++ output ++ "\n") time) model
```

Reply:
Wait, based on the discussion with @vito above (https://github.com/concourse/concourse/pull/5846#issuecomment-665121993), it looks like the decision was to ignore this information in the web UI and only log a message in `fly watch`

Reply:
Hmm, yeah that was the idea, but I was more assuming that @evanchaoli wouldn't be interested in doing any UI work so I suggested that we skip it for now.

My main concern was around the way the data is recorded. Now that it has its own event type, we can at least change the UI in the future, so I'm OK with doing this for now.

Reply:
I don't know ELM at all, so I am not able to contribute UI part.

The current implementation has met my needs, so it would relief pressure from your side, as I'll no longer push for UI part. 😄